BaragonService/src/main/java/com/hubspot/baragon/service/worker/BaragonRequestWorker.java

```diff
@@ -190,6 +189,8 @@ public void run() {
           }
         }
       }
+
+      requestManager.failStuckPendingRequests();
```

Comment:
was thinking similar after our conversation about the zk data structure change. Let me see how hard it would be to do that all as a transaction. That would alleviate the error without a need for extra checks each time